media/inf/shortcuts.inf

```diff
@@ -1,4 +1,4 @@
-﻿[Version]
+[Version]
```

Comment:
Some unrelated change here, you probably inserted UTF-8 BOM.

Reply:
Reverted